src/Function/Construct/Symmetry.agda

```diff
@@ -11,11 +11,13 @@ module Function.Construct.Symmetry where
 open import Data.Product.Base using (_,_; swap; proj₁; proj₂)
 open import Function.Base using (_∘_)
 open import Function.Definitions
-  using (Bijective; Injective; Surjective; Inverseˡ; Inverseʳ; Inverseᵇ; Congruent)
+  using (Bijective; Injective; Surjective; Inverseˡ; Inverseʳ; Inverseᵇ
+        ; Congruent)
 open import Function.Structures
   using (IsBijection; IsCongruent; IsRightInverse; IsLeftInverse; IsInverse)
 open import Function.Bundles
-  using (Bijection; Equivalence; LeftInverse; RightInverse; Inverse; _⤖_; _⇔_; _↩_; _↪_; _↔_)
+  using (Bijection; Equivalence; LeftInverse; RightInverse; Inverse; _⤖_; _⇔_
+        ; _↩_; _↪_; _↔_)
```

Comment:
```suggestion
  using (Bijection; Equivalence; LeftInverse; RightInverse; Inverse
        ; _⤖_; _⇔_; _↩_; _↪_; _↔_)
```
grouping: the general `Setoid` definitions on one line, the `Propositional` specialisations on the next?